wrench/reftests/performance/no-clip-mask.yaml

```diff
@@ -0,0 +1,19 @@
+---
+root:
+  items:
+    -
+      type: "stacking-context"
+      items:
+        -
+          bounds: [0, 111, 1887, 1365]
+          "clip-rect": [0, 111, 1887, 1365]
```

Comment:
It might be worth leaving a comment here explaining why this test doesn't generate any clip masks.